src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/02 Advanced Examples.md

```diff
@@ -3,11 +3,19 @@ title: 'Advanced Examples'
 excerpt: 'Advanced Examples using the k6 Scenario API - Using multiple scenarios, different environment variables and tags per scenario.'
 ---
 
-## Using multiple scenarios
+You can use multiple scenarios in one script, and these scenarios can be run in sequence or in parallel.
+Some ways that you can combine scenarios include the following:
+- Have different start times to sequence workloads
+- Add per-scenario tags and environment variables
+- Make scenario-specific thresholds.
+Use multiple scenarios to run different test logic, so that VUs don't run only the `default` function](https://k6.io/docs/using-k6/test-lifecycle/).
```

Comment:
```suggestion
 - Use multiple scenarios to run different test logic, so that VUs don't run only the `default` function](https://k6.io/docs/using-k6/test-lifecycle/).
```